sys/net/application_layer/dhcpv6/client.c

```diff
@@ -131,20 +147,61 @@ static void *_thread(void *args)
     event_queue_t event_queue;
     event_queue_init(&event_queue);
     dhcpv6_client_init(&event_queue, SOCK_ADDR_ANY_NETIF);
-    /* TODO: add configuration for IA_NA here, when implemented */
     dhcpv6_client_start();
     event_loop(&event_queue);   /* never returns */
     return NULL;
 }
 #endif /* MODULE_AUTO_INIT_DHCPV6_CLIENT */
 
+void _print_ia_na_debug_info(uint16_t netif, int result_code)
+{
+    if (result_code == 0) {
+        return;
+    } else if (result_code == -ENOMEM) {
```

Comment:
```suggestion
    } else {
```

should suffice with a corresponding update to the DEBUG message. When DEBUG is enabled, the user should know by now, that IA_NA is not enabled, so they can draw their own conclusion what the problem may and we safe a check ;-).